code/modules/reagents/chemistry/recipes/others.dm

```diff
@@ -543,6 +543,23 @@
 		new /mob/living/simple_animal/pet/dog/corgi(location)
 	..()
 
+//monkey powder heehoo
+/datum/chemical_reaction/monkey_powder
+	name = /datum/reagent/monkey_powder
+	id = /datum/reagent/monkey_powder
+	results = list(/datum/reagent/monkey_powder = 3)
+	required_reagents = list(/datum/reagent/consumable/banana = 1, /datum/reagent/consumable/nutriment=2,/datum/reagent/liquidgibs = 1)
+
+/datum/chemical_reaction/monkey
+	name = "monkey"
+	id = "monkey"
+	required_reagents = list(/datum/reagent/monkey_powder = 30, /datum/reagent/water = 1)
+
```

Comment:
```suggestion
    mob_react = FALSE
    
```
if you dont want players shitting out monkeys

Reply:
I kind of like players being able to give birth to monkeys via a reaction in their body.
Willing to accept input though.

Reply:
same